css-brochure-form/styles.css

```diff
@@ -0,0 +1,67 @@
+body {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+h2 {
+  font-family: "Brandon Grotesque", sans-serif;
+  color: #e34f3d;
+  font-weight: 500;
+  font-size: 15pt;
+  text-transform: uppercase;
+  letter-spacing: 1px;
+}
+
+label {
+  font-family: "Brandon Grotesque", sans-serif;
+  font-weight: 500;
+  font-size: 10pt;
+  text-transform: uppercase;
+  margin-left: 0.5rem;
+}
+
+input {
+  width: 450px;
+  height: 40px;
+  margin-bottom: 1rem;
+  margin-top: 3px;
+  border-radius: 5px;
+  border: 1px solid;
+  border-color: lightgray;
+}
+
+input::placeholder {
```

Comment:
Let's move all these stylings to `input` because in that way when user clicks on input, any typing that do also have same font, font size and paddings. But let's remove font color because by default, placeholder will have its own gray color.
![Screenshot 2022-09-02 at 1 48 01 PM](https://user-images.githubusercontent.com/9010389/188233336-9f3d190c-02cb-4e19-a36b-e7de398906a9.png)